langchain/callbacks/manager.py

```diff
@@ -398,6 +402,55 @@ async def on_tool_error(
         )
 
 
+def _configure(
+    callback_manager_cls: Type[CallbackManager] | Type[AsyncCallbackManager],
+    inheritable_handlers: Optional[
+        Union[BaseCallbackManager, List[BaseCallbackHandler]]
+    ] = None,
+    local_handlers: Optional[
+        Union[BaseCallbackManager, List[BaseCallbackHandler]]
+    ] = None,
+    verbose: bool = False,
+) -> Optional[BaseCallbackManager]:
+    callback_manager: Optional[BaseCallbackManager] = None
+    if inheritable_handlers or local_handlers:
+        if isinstance(inheritable_handlers, list) or not inheritable_handlers:
+            callback_manager = callback_manager_cls(
```

Comment:
This block is confusing to me on the second condition. "If not inheritable_handlers" -> then this would be equivalent to `callback_manager_cls()` with no args?